### ISSUE TITLE
Audio slowdown on live tv

### ISSUE THREAD

**Issue by HeresJonny** [xbmc/xbmc#15617]
### Describe the bug
Here is a clear and concise description of what the problem is:

Audio slowdown to half (?) speed in live tv stream (IPTV simple). Notebook 60 fps screen. Behaviour is new in KodiSetup-20190224-7e853c72-master-x64.exe, was not present in 20190222.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

### To Reproduce

1. 
2.
3.

### Debuglog

The debuglog can be found here:

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Raspberry-Pi
 - [X] Windows x64
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version:
KodiSetup-20190224-7e853c72-master-x64.exe

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by DaVukovic** [comment]
We will at least need a debug log for that. Hence we ask for it on our issue template. 

Please provide it. I'll mark the issue as invalid for the time being. 

Thanks in advance.

**Comment by HeresJonny** [comment]
Sorry, forgot to specify: To reproduce, turn on "Sync playback to display".

**Comment by fritsch** [comment]
Before https://github.com/xbmc/xbmc/pull/15586 Sync Playback to Display was broken. Means you might now get the intended behaviour of this setting. Your debuglog will tell how large the speedup is, so please post it.

@peak3d for info

**Comment by peak3d** [comment]
@HeresJonny pls. provide a debug log

**Comment by HeresJonny** [comment]
https://pastebin.com/ACtpP9Vi

**Comment by fritsch** [comment]
```
2019-02-26 09:41:28.570 T:5580   DEBUG: CVideoReferenceClock: Clock speed 50.000000%
```
:-)

**Comment by FernetMenta** [comment]
> Means you might now get the intended behaviour of this setting

really? :)

**Comment by HeresJonny** [comment]
Looks like the display runs at 50 fps and not 60 fps, sorry... and also not 50% :-)

**Comment by peak3d** [comment]
simply some logical error :-)

**Comment by fritsch** [comment]
@FernetMenta hehe - obviously not :-)

Next time I'll ask @peak3d directly when he changes some tuning values that are not self explaining ...

**Comment by peak3d** [comment]
@HeresJonny you can try this build: https://mirrors.kodi.tv/test-builds/windows/win64/KodiSetup-20190226-bcf5ec2a-refclock-x64.exe

**Comment by BytEvil** [comment]
Same issue on linux.

**Comment by peak3d** [comment]
This issue affects all platforms, no need to list all different platforms here, thx!

**Comment by DaVukovic** [comment]
Fixed by #15619 

Closed
